docs/tbdev_getting_started.ipynb

```diff
@@ -12,7 +12,7 @@
     },
```

Comment:
<p>Similar to the above comment, although having one line to say the TensorBoard callback helps log to TensorBoard and the hp.KerasCallback logs the hparams seems fine. I don't think we need to mention the histogram_freq parameter and for the HParams link can just say "learn more <a href="https://www.tensorflow.org/tensorboard/hyperparameter_tuning_with_hparams" target="_blank">here</a>". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3906/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I simplified it. Can you give it another look?